haskell/src/Ligo/BaseDAO/TZIP16Metadata.hs

```diff
@@ -39,6 +42,16 @@ data MetadataSettings = MetadataSettings
   { msConfig :: MetadataConfig
   }
 
+-- | TZIP-21 assets field.
+data MetadataAsset = MetadataAsset
+  { maName :: Text
+  , maDecimals :: Integer
+  , maDescription :: Text
+  , maShouldPreferSymbol :: Bool
+  , maThumbnailUri :: Text
+  , maCreators :: [Text]
```

Comment:
It seems that this and `mdDescription` are suggested in `TZIP-21` for [Semi-fungible tokens and NFTs](https://gitlab.com/tezos/tzip/-/blob/master/proposals/tzip-21/tzip-21.md#semi-fungible-and-nft-token-recommendations), given that frozen tokens are fungible I would remove both.

One reason is that their presence might be confusing to indexers and other tools, the other is that I'm not sure that `creators` should be the same as the contract authors (this field especially seems to be more suited for unique tokens on the blockchain).